Hw note app

StasonLV opened the pull request:
Приложение -заметка

StasonLV committed:
Добавлены текстфил и текствью, бэкграунд

StasonLV committed:
Added funcionality

Добавлена кнопка "Сэйв", возможность сохранять контент по нажатию этой кнопки, закрытие клавиатуры по нажатию на копку"Сэйв", по нажатию вне поля ввода, добавлен плейсхолдер для заголовка, загрузочный экран.

bsciosedu commented on Trainee app/SavingData.swift:
```diff
@@ -0,0 +1,49 @@
+        case noteData
+    }
+    
+    static var titleData: String! {
```
Давай избавимся тут от static и force unwrapping, если хочешь хранить данные сейчас то лучше пока что это сделать в ViewControllere

StasonLV replied:
Вплане не использовать юзердифолтс для хранения?

bsciosedu replied:
Почему, использовать можно просто сделать это без static и force unwrapping

StasonLV replied:
А, окей, понял, сегодня доработаю

StasonLV committed:
Доделанная ДЗ

Изменен метод сохранения в юзер дифолтс, сделан навигейшн контроллер, добавлен таб бар айтем, отвечающий за сохранение, удалена лишняя кнопка сохранения.

StasonLV committed:
Устранение замечаний от 29.03.22

Поправлены отступы, сделана структура с константами для текстов, ключей(вместо енама), тайтла таббара. Текствью привязана к низу вью, добавлена прокрутка. Исправлены методы, чтобы не перегружать вьюдидлоад.

StasonLV committed:
Исправлены имена методов сохранения и выгузки даты

StasonLV committed:
Swiftlint

Добавил swiftlint, отредактировал код в соотв. с ошибками линта.

StasonLV committed:
Класс определен как final

StasonLV commented:
ДЗ номер один
